Xamarin.Forms.Platform.Android/CollectionView/CarouselViewRenderer.cs

```diff
@@ -64,6 +64,8 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateIsBounceEnabled();
 			else if (changedProperty.Is(LinearItemsLayout.ItemSpacingProperty))
 				UpdateItemSpacing();
+			else if (changedProperty.Is(ItemsView.ItemsUpdatingScrollModeProperty))
+				UpdateItemsUpdatingScrollMode();
```

Comment:
This is an `ItemsView` property. Shouldn't it be updated in `ItemsViewRenderer`? Same with line 57. @rmarinho